src/geophires_x/CylindricalReservoir.py

```diff
@@ -92,13 +96,14 @@ def __init__(self, model: Model):
         )
         self.RadiusOfEffectFactor = self.ParameterDict[self.RadiusOfEffectFactor.Name] = floatParameter(
             "Cylindrical Reservoir Radius of Effect Factor",
+            value=1.0,
```

Comment:
Remove - only `DefaultValue` should be specified, not `value` (which is redundant with `DefaultValue` in this case anyway)